dateparser_cli/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class FastTextModelNotFoundException(Exception):
+    def __init__(self, message):
+        self.message = message
+        super().__init__(self.message)
+
+
+class CommandNotFound(Exception):
+    def __init__(self, message):
+        self.message = "dateparser-download: {}".format(message)
+        super().__init__(self.message)
```

Comment:
I think these two can be simplified to

```
class FastTextModelNotFoundException(Exception):
    pass
```

and same for the other one, what do you think?

Reply:
done 👍🏻